src/components/App.jsx

```diff
@@ -1,8 +1,8 @@
-import { Routes, Route } from "react-router-dom";
+import { Route, Routes } from "react-router-dom";
 import { Reset } from "styled-reset";
 
 import Onboarding from "./Onboarding";
-import NewProejct from "./NewProject";
+import NewProject from "./NewProject";
```

Comment:
```suggestion
// import 외부 라이브러리 (react와 관련된 라이브러리부터 ~ 스타일 라이브러리 순으로)

// import 컴포넌트

// import 유틸함수
```
저는 이렇게 3묶음으로 import를 작성하는 편인데 혹시 어떠신가요?

Reply:
해당 방식으로 통일하겠습니다.